spec/unit/util/windows/sid_spec.rb

```diff
@@ -99,6 +99,7 @@
         # Ruby seems to use the local codepage when making COM calls
         # if this fails, might want to use Windows API directly instead to ensure bytes
         user = Puppet::Util::Windows::ADSI.create(username, 'user')
+        user.SetPassword('PUPPET_RULeZ_123!')
```

Comment:
it's from a different commit, I'm going to rebase my branch that this PR is based off of so it goes away